test/dummy/spec/controllers/ish_manager/galleries_controller_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'spec_helper'
-
+require 'rails_helper'
```

Comment:
it already requires spec_helper. Require rails_helper from spec_helper if you really need it, but really you should be using spec_helper only.